### PR TITLE
Update content less standards

### PR DESCRIPTION
**Pull request by mrtuvn**
## Purpose of this pull request

This pull request (PR) update content less related mixins folder to avoid confuse and misleading while theme less development

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-less.html
https://devdocs.magento.com/guides/v2.4/coding-standards/code-standard-less.html
## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.3/coding-standards/code-standard-less.md [hunk 1]
```diff
@@ -782,3 +782,6 @@ Use single quotes.
 ```css
 @import "source/lib/_lib.less";
 ```
+
+<!-- Link definitions -->
+[Css Theme]: {{ site.baseurl }}/guides/v{{ site.version }}/frontend-dev-guide/css-topics/css-themes.html
```
Comment (BarnyShergold) [inline src/guides/v2.3/coding-standards/code-standard-less.md:787]:
```suggestion
[Css Theme]: {{ page.baseurl }}/frontend-dev-guide/css-topics/css-themes.html
```

File: src/guides/v2.3/coding-standards/code-standard-less.md [hunk 2]
```diff
@@ -720,7 +720,7 @@ Component name must meaningful. It can contain the `primary`, `secondary`, `tert
 
 ### Location
 
-[Theme](https://glossary.magento.com/theme) mixins (except extends) should be located in the `source/utilities` directory.
+[Theme](https://glossary.magento.com/theme) mixins (except extends) should be located in the `web/css/source` directory. For more details, refer to our [Css Theme][].
```
Comment (BarnyShergold) [inline src/guides/v2.3/coding-standards/code-standard-less.md:723]:
```suggestion
[Theme](https://glossary.magento.com/theme) mixins (except extends) should be located in the `web/css/source` directory. For more details, refer to [Css Theme][].
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
Good to go, @BarnyShergold ?

**Review by BarnyShergold — APPROVED** [review]
Good to go!

**Comment by dobooth** [comment]
running tests

**Pull request merged**